admin/partials/nginx-helper-general-options.php

```diff
@@ -604,9 +646,13 @@
 						echo wp_kses(
 							sprintf(
 								'%1$s<br /><small>%2$s</small>',
-								esc_html__( 'Nginx Map path to include in nginx settings', 'nginx-helper' ), esc_html__( '(recommended)', 'nginx-helper' )
+								esc_html__( 'Nginx Map path to include in nginx settings', 'nginx-helper' ),
+								esc_html__( '(recommended)', 'nginx-helper' )
```

Comment:
This is not correct way to make small string translatable. We need to allow entire string translatable.